projects/llm-d/testing/test_llmd.py

```
@@ -834,6 +983,235 @@ def apply_resource_configuration(isvc_data, model_key):
+                             capture_stdout=True, check=True)
+            if not result.stdout.strip():
+                # Extract just the filenames from hotfix_files for the error message
+                filenames = [file_path.split('/')[-1] for file_path in hotfix_files]
+                file_args = ' \\\n       '.join([f"--from-file=guides/pd-disaggregation/ms-pd/charts/vllm-ucx-multiproc-hotfix/{filename}" for filename in filenames])
+
+                raise RuntimeError(f"Required ConfigMap '{cm_name}' not found in namespace '{namespace}'. "
+                                   f"Please create it first:\n\n"
+                                   f"oc create cm vllm-ucx-multiproc-hotfix -n {namespace} \\\n"
+                                   f"       {file_args}")
+            logging.info(f"Verified ConfigMap '{cm_name}' exists in namespace '{namespace}'")
+        except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)